### ISSUE TITLE
Clicking on `Show all tokens` on the address page results in a scroll to the top

### ISSUE THREAD

**Issue by yvaskov** [blockscout/frontend#2433]
## Description

Clicking on `Show all tokens` (wallet icon) on the address page results in the scroll to the `Tokens` tab and an immediate scroll back to the top. Reproducible on desktop and mobile when any other tab is open, rarely happens with the `Tokens` tab open.

https://github.com/user-attachments/assets/93557b8b-d58f-4859-b0ad-4f856c6a2d59

## Version

[v1.36.3](https://github.com/blockscout/frontend/tree/v1.36.3)

**Comment by Rjsoon38** [comment]
> ## Description
> 
> Clicking on `Show all tokens` (wallet icon) on the address page results in the scroll to the `Tokens` tab and an immediate scroll back to the top. Reproducible on desktop and mobile when any other tab is open, rarely happens with the `Tokens` tab open.
> 
> https://github.com/user-attachments/assets/93557b8b-d58f-4859-b0ad-4f856c6a2d59
> 
> ## Version
> 
> [v1.36.3](https://github.com/blockscout/frontend/tree/v1.36.3)

[NOTICE] Bootstrapped 10% (conn_done): Connected to a relay[NOTICE] Closing no-longer- listener on ???:0[NOTICE] DisableNetwork is0x5283d291dbcf85356a21ba090e6db59121208b44


0x5283d291dbcf85356a21ba090e6db59121208b44

- [ ] tourist eviI dетаiI аwfuI snaск сlaрBlockscan Chat wants you to sign in with your web3 account
0xdc6999513539883ee37f4f1a0a2ad573812b6a68

URI: https://chat.blockscan.com
Version: 1
Chain ID: ALL
Nonce: hjd3uen8
Issued

**Comment by isstuev** [comment]
@yvaskov I can't reproduce, can you?

**Comment by yvaskov** [comment]
@isstuev yep: https://eth-sepolia.k8s-dev.blockscout.com/address/0x4281eCF07378Ee595C564a59048801330f3084eE?tab=txs

https://github.com/user-attachments/assets/bfb5c73f-19d0-4def-841e-a70ef37c743d

**Comment by isstuev** [comment]
tabs were moved to the top of the page, can't reproduce with the current layout
